src/components/menu/mobile-menu.js

```diff
@@ -83,6 +83,33 @@ export const MobileMenuLink = styled(Link)`
     }
 `
 
+export const ExternalMobileMenuLink = styled.a`
```

Comment:
i don't think the MobileMenuLink definition needs to be repeated for ExternalMobileMenuLink.